### PR TITLE
NAS-124299 / 24.04 / Removing table service and moving method to component

### PR DESCRIPTION
**Pull request by RehanY147**
Check ticket description for details

### CODE REVIEW

File: src/app/pages/dashboard/components/widget-network/widget-network.component.html [hunk 1]
```diff
@@ -100,7 +100,10 @@ <h4 class="info-header-title">{{ nic.state.name }}</h4>
                         </li>
                         <li fxLayout="row" fxLayoutAlign="start center" class="line-state">
                           <span class="icon state">
-                            <ix-icon
+                            <ix-interface-status-icon
+                              [update]="networkInterfaceUpdate.get(nic.state.name)"
+                            ></ix-interface-status-icon>
+                            <!-- <ix-icon
                               *ngIf="getLinkState(nic) === LinkState.Up"
```
Comment (AlexKarpov98) [inline src/app/pages/dashboard/components/widget-network/widget-network.component.html:107]: I think we can remove this zombie comment:
Screenshot 2023-10-13 at 14 42 43

File: src/app/modules/common/interface-status-icon/interface-status-icon.component.ts [hunk 2]
```diff
@@ -39,11 +42,25 @@ export class InterfaceStatusIconComponent implements OnChanges {
...
+  updateStateInfoIcon(type: 'sent' | 'received'): void {
+    if (!this.stateIcon) {
+      return;
+    }
+    const arrowIcons = this.stateIcon._elementRef.nativeElement.querySelectorAll('.arrow');
+    const targetIconEl = type === 'sent' ? arrowIcons[0] : arrowIcons[1];
+
+    targetIconEl.classList.add('active');
+
+    timer(2000).pipe(untilDestroyed(this)).subscribe(() => {
```
Comment (AlexKarpov98) [inline src/app/modules/common/interface-status-icon/interface-status-icon.component.ts:62]: I think we can add `take(1)` to ensure this one is taken only once.

**Review by AlexKarpov98 — APPROVED** [review]
Looks good, thanks.

File: src/app/modules/common/interface-status-icon/interface-status-icon.component.ts (hunk 2 above)
Comment (RehanY147) [reply src/app/modules/common/interface-status-icon/interface-status-icon.component.ts:62]: The timer operator only emits one so no need here.

**Pull request merged**